aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1749,7 +1755,7 @@ def _call_route(self, route: Route, route_arguments: Dict[str, str]) -> Response
             # Reset Processed stack for Middleware (for debugging purposes)
             self._reset_processed_stack()
 
-            return ResponseBuilder(
+            return self.response_builder_class(
```

Comment:
same as before - is there a reason why we are allocating extra memory to have `ResponseBuilder` into a public attribute?